src/lightning_app/core/work.py

```diff
@@ -272,7 +272,7 @@ def statuses(self) -> List[WorkStatus]:
     @property
     def has_started(self) -> bool:
         """Return whether the work has started."""
-        return self.status.stage != WorkStageStatus.NOT_STARTED
+        return self.status.stage == WorkStageStatus.STARTED
```

Comment:
Can you explain this change? I don't think it is obvious. In the past, we struggled defining the negation of started and not started etc.

Reply:
Removed.